hasher/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ruby:alpine
+FROM ruby:3-alpine3.13
```

Comment:
How specific do we have to be for this one? Should we be consistent with `python`, and use `ruby:3-alpine` here?

Reply:
I put alpine version here because there is some apk commands in the Dockerfile.
It's not obvious that those packages will be the same in let's say alpine4.

For the other ones I didn't pin alpine because there is no system dependencies in the Dockerfiles, nor in the sources.

But if you prefer/ it's clearer, I can pin alpine everywhere.

Reply:
My thinking was that this is generally a "low activity" repository, and that there's a higher risk that a future "ruby 3" image will no longer have an alpine 3.13 variant, than that alpine would no longer have a package

Mostly trying to reduce maintenance